app/src/main/java/com/example/fitnesspage/FitnessPlanApp.kt

```diff
@@ -11,11 +12,20 @@ import androidx.compose.runtime.collectAsState
 import androidx.lifecycle.viewmodel.viewModelFactory
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
-import com.example.fitnesspage.ui.theme.SurveyScreen
+//import com.example.fitnesspage.ui.theme.SurveyScreen
```

Comment:
A minor comment about leaving commented code behind.  They get on me at work about this.

Reply:
duly noted, thanks.